apps/web/src/services/asset-inventory/types/collector-main-page-type.ts

```diff
@@ -54,6 +40,7 @@ export interface CollectorItemInfo {
     detailLink: CollectorLink;
     schedule?: Schedule;
     recentJobAnalyze?: JobAnalyzeStatus[];
+    resourceGroup: ResourceGroupType;
```

Comment:
suggestion: If this doesn't contain all resource groups(SYSTEM, DOMAIN, WORKSPACE, PROJECT), please use `Extract<>`.